enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -223,27 +218,23 @@ func initializeSociStoreKeychainClient(env environment.Env, target string) (sspb
 	return sspb.NewLocalKeychainClient(conn), nil
 }
 
-func (p *Provider) NewContainer(ctx context.Context, image string, options *PodmanOptions) (container.CommandContainer, error) {
-	imageIsStreamable := false
-	for _, streamableImage := range p.streamableImages {
-		if image == streamableImage {
-			imageIsStreamable = true
-			break
-		}
+func (p *Provider) NewContainer(ctx context.Context, image string, imageIsPublic bool, options *PodmanOptions) (container.CommandContainer, error) {
+	var imageIsStreamable bool
+	if imageIsPublic {
+		imageIsStreamable = *imageStreamingEnabled
```

Comment:
nit: instead of directly depending on the flag value in 2 places, you can set a Provider property (imageStreamingEnabled) in NewProvider and reference that in NewContainer.

Reply:
Done